### DISCUSSION TITLE
Difference between --policy and --config-policy

### DISCUSSION THREAD

**Discussion by AnaisUrlichs** [aquasecurity/trivy#4004] [Q&A] [answered]
I am not sure I understand the difference between each?

<img width="942" alt="Screenshot 2023-04-06 at 14 54 56" src="https://user-images.githubusercontent.com/33576047/230399451-5f58cf82-6acf-4fff-88ec-1158f0b5bb71.png">

**Comment by itaysk** [comment, marked as answer]
looks like `--policy` is alias of `--config-policy` https://github.com/aquasecurity/trivy/blob/main/pkg/flag/rego_flags.go#L28
so they're the same as you've observed
